docs/rancher.md

```diff
@@ -0,0 +1,320 @@
+How to use TopoLVM with Rancher/RKE
+===================================
+
+This document is a brief introduction of how to use TopoLVM on [Rancher/RKE](https://rancher.com/docs/rke/latest/en/).
```

Comment:
"gcloud" command (and initialize the command) is required to try this guide.
So please write the requirement.